### PR TITLE
[explore] Display topic image along with title and description.

### PR DESCRIPTION
**Pull request by keyurva**
* Displays topic image along with title and description per mocks.
* Currently all topics will show the health image.
   + Will be adding others once Adriana has had a chance to export them.
 
[explore-image.webm](https://github.com/datacommonsorg/website/assets/1221814/e00b6119-702d-4745-948d-3a0b24778688)

### CODE REVIEW

File: static/js/apps/explore_landing/topics.json [hunk 1]
```diff
@@ -715,6 +721,7 @@
     "sdg": {
       "title": "Sustainability Development Goals",
       "description": "Explore global and country level progress on the 17 Sustainable Development Goals (SDGs).",
+      "image": "/images/topics/health.svg",
```
Comment (dwnoble) [inline static/js/apps/explore_landing/topics.json:724]: does this file need to be checked in?

Comment (keyurva) [reply static/js/apps/explore_landing/topics.json:724]: Argh. Of course - thanks. Done.

**Review by pradh — APPROVED** [review]

**Comment by pradh** [comment]
Keyur, I approved/sync'd this to get it in, so I can do follow on changes.

**Review by dwnoble — COMMENTED** [review]
Nice!

**Pull request merged**